components/applied-filters/applied-filters.js

```diff
@@ -7,6 +7,23 @@ import { getComposedChildren } from '@brightspace-ui/core/helpers/dom';
 import { LocalizeStaticMixin } from '@brightspace-ui/core/mixins/localize-static-mixin.js';
 import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
 
+/* eslint sort-imports: "off" */
+import arSa from './lang/ar-sa.js';
+// import de from './lang/de.js';
+import en from './lang/en.js';
+// import es from './lang/es.js';
+import frCa from './lang/fr-ca.js';
```

Comment:
Is this component going to have fr-ON translations as well?

Reply:
I had that question as well. Jeff said that this component doesn't require fr-ON translations 👍